Core/Config.cpp

```diff
@@ -693,6 +693,8 @@ static const ConfigSetting graphicsSettings[] = {
 
 	ConfigSetting("UberShaderVertex", &g_Config.bUberShaderVertex, true, CfgFlag::DEFAULT),
 	ConfigSetting("UberShaderFragment", &g_Config.bUberShaderFragment, true, CfgFlag::DEFAULT),
+
+	ConfigSetting("DisplayRefreshRate", &g_Config.iDisplayRefreshRate, g_Config.iDisplayRefreshRate, CfgFlag::PER_GAME),
```

Comment:
This is a bit strange tho, the issue probably at `framerate = g_Config.iDisplayRefreshRate;` which should be set during `void __DisplayInit()` i think.

Reply:
Which issue ?

Reply:
the failed tests thing

Reply:
Ah, they were already fixed by https://github.com/hrydgard/ppsspp/pull/19325/commits/1aec09a056b616034677f106e2450c8833a083b2 :)

Reply:
Yes, but it doesn't seems like a proper fix (which is why i commented there too), if taking account of PPSSPP.ini

Have you tried changing the value at PPSSPP.ini and see whether that value being shown correctly on the UI the next time you ran PPSSPP ?

Reply:
See https://github.com/hrydgard/ppsspp/pull/19325#discussion_r1677073603

Reply:
The config value doesn't even have a default by the time `__DisplayInit()` gets to run, so `framerate = 0` and tests break.

There's no issues initialising it in the header at compile-time, because it used to be a constant anyway and we know monitors typically target 60.

Tests and runtime work perfectly currently, I think we'd have to make the config get initialised before the display code runs, which would require more code changes.

Reply:
I just did some printf: the headless environment differs from SDL.

In a headless test, it's always `g_Config.iDisplayRefreshRate = 0`, but in SDL, `g_Config.iDisplayRefreshRate = 165` from the very first frame, so.

So, it's only during tests that ConfigSetting constructors don't run. In SDL, this runs before the display code.

What do we do ?

Reply:
Not sure why it shows different value, and i'm not familiar with the headless test either :)